### PR TITLE
fix: Add safeguard for drop provider

### PR DESCRIPTION
**Pull request by disq**

### CODE REVIEW

File: cmd/provider.go [hunk 1]
```diff
@@ -117,6 +122,7 @@ func init() {
 		"last-update is the duration from current time we want to remove resources from the database. "+
 			"For example 24h will remove all resources that were not update in last 24 hours. Duration is a string with optional unit suffix such as \"2h45m\" or \"7d\"")
 	providerRemoveStaleCmd.Flags().BoolVar(&dryRun, "dry-run", true, "")
+	providerDropCmd.Flags().BoolVar(&iAmSure, "i-am-sure", false, "Really drop tables for the provider")
```
Comment (roneli) [inline cmd/provider.go:125]: nit: --force ? --approve-drop ?

**Review by roneli — APPROVED** [review]
LGTM had 1 nit.

File: cmd/provider.go (hunk 1 above)
Comment (bbernays) [reply cmd/provider.go:125]: I like --force

Comment (disq) [reply cmd/provider.go]: Done!

Comment (disq) [reply cmd/provider.go]: ...but `--force` implies "it's normally possible to do it without forcing, but for some reason, in this instance, we can't" which is not the case here.

**Review by bbernays — APPROVED** [review]

**Pull request merged**